_health-care/_js/components/form-elements/ErrorableCheckbox.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import _ from 'lodash';
+
+/**
+ * A form checkbox with a label that can display error messages.
+ *
+ * Validation has the following props.
+ * `errorMessage` - Error string to display in the component.
+ *                  When defined, indicates checkbox has a validation error.
+ * `checkedValue` - String. The value that should be saved when checked.
+ * `label` - String for the checkbox label.
+ * `required` - boolean. Render marker indicating field is required.
+ * `value` - string. Value of the checkbox.
+ * `onValueChange` - a function with this prototype: (newValue)
+ */
+class ErrorableCheckbox extends React.Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.inputId = _.uniqueId('errorable-checkbox-');
+  }
+
+  handleChange() {
```

Comment:
I don't quite fully follow this.

Checkboxes in HTML for a Form submit do indeed have a funky bit of logic where if they aren't checked, nothing is sent to the server on a form submit.

But we don't actually have a need to be bound to those rules.

I think we should be able to just inspect the `checked` attribute on the DOM node and not use the `value` attribute at all.